### ISSUE TITLE
LARD and TRIM ? 

### ISSUE THREAD

**Issue by amvarani** [oushujun/EDTA#214]
Hi there
EDTA pipeline is capable to pick-up LARDs and TRIMs elements  ?

**Comment by oushujun** [comment]
Yes these can be found in the intact LTR list but you will need to pick
them out yourself based on their definition. - Shujun

**Comment by amvarani** [comment]
Okay,
Another question: the pipeline keep these in the final annotation files ? 

Thanks for your reply!

**Comment by oushujun** [comment]
of course! These belong to LTR elements. They were just not labeled out.
-Shujun
